### PR TITLE
Add language event handling

### PR DESCRIPTION
**Pull request by toshisasaki**
The user can set the language.  When a room is created, the room
language is set and only users with the same language set can join.

Signed-off-by: Anderson Toshiyuki Sasaki <anderson.sasaki@gmail.com>

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -127,6 +116,16 @@ <h5 id="gameInstructions"></h5>
...
+            <div class="rooms">
+                <div class="half">
+                    <h4 id="room" align="center">Room: Lobby</h4>
+                    <ul id="roomPlayers" align="center" width="80%"></ul>
+                </div>
+                <div class="half">
+                    <h4 id="gameID" align="center">Game: not in a game</h4>
+                    <ul id="gamePlayers" align="center" width="80%"></ul>
+                </div>
+            </div>
```
Comment (yuumasato) [inline public/index.html:128]: Any special reason for this change?

I think I preferred when the table was at the top.

File: src/events/language.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const Users = require('../server/users.service');
+const consts = require('../server/consts');
+
+// Initialize event listener
+module.exports = function(socket) {
+    socket.on('language', (language) => {
+
+        // TODO emit error
+        if (!consts.SUPPORTED_LANGUAGES.includes(language)) {
+            console.error(`Language ${language} not supported`);
```
Comment (yuumasato) [inline src/events/language.js:11]: I assume the errors will sent back to the client will be implemented later

**Review by yuumasato — APPROVED** [review]
LGTM, I have just one question about the testing interface.

**Comment by yuumasato** [comment]
Tested with https://github.com/yuumasato/have_you_heard/pull/65

File: public/index.html (hunk 1 above)
Comment (toshisasaki) [reply public/index.html:128]: Ok, I'll move them back to the top.

File: src/events/language.js (hunk 2 above)
Comment (toshisasaki) [reply src/events/language.js:11]: Yes, this part is not ready yet

**Review by yuumasato — APPROVED** [review]

**Pull request merged**